assets/js/src/components/ResponsiveImage.jsx

```diff
@@ -132,7 +134,7 @@ class ResponsiveImage extends React.Component {
       loadImage,
       onLoad,
       preloadBackground,
-      ref: 'currentImageSize',
+      ref: cis => (this._currentImageSize = cis),
```

Comment:
If you wouldn't mind, could the `_` get removed from the beginning of the ref variable name? Just for consistency because no other variables use that naming convention.